src/main/java/uk/gov/ida/matchingserviceadapter/validators/DateTimeComparator.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.matchingserviceadapter.validators;
+
+import org.joda.time.DateTime;
+import org.joda.time.Duration;
+
+public class DateTimeComparator {
```

Comment:
How does this compare (excuse the pun) with org.joda.time.DateTimeComparator?

Reply:
I like the explicit nature of our comparator

eg DateTimeComparator#isAfterNow